lib/auth/join.go

```diff
@@ -162,7 +166,8 @@ func (a *Server) generateCerts(ctx context.Context, provisionToken types.Provisi
 			renewable = true
 		case types.JoinMethodIAM,
 			types.JoinMethodGitHub,
-			types.JoinMethodCircleCI:
+			types.JoinMethodCircleCI,
+			types.JoinMethodKubernetes:
```

Comment:
possibly this should be removed if tbot support is coming out of this PR, it might be more clear to have everything in an eventual tbot support PR